### PR TITLE
fix (DB/NPC): Dalaran Visitor CTM fix

### PR DESCRIPTION
**Pull request by acidmanifesto**
Fixes the air swimmers when they land.

## Changes Proposed:
-  Deletes CTMS of two npcs the Dalaran Visitors
-  Smart sets their inhabit type and they are just a quick 8 second skit before despawning.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/10055

## SOURCE:

- Visual comaprisons as they are none flying humanoids.

## Tests Performed:

- Builds
- Performs

## How to Test the Changes:

1. goto krauses landing in dalaran
2. watch the dalaran visitors land
3. you will be looking at a gnome and a dwarf dalaran vistor as the culprits. they now run as intended per the smart scripts.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by IntelligentQuantum — CHANGES_REQUESTED** [review]
remove acore_world

**Comment by acidmanifesto** [comment]
> remove acore_world
> 
> and `CreatureId` IN(32596, X); maybe

Ah. Was a quick pr before i left for my appointment currently. Will definately do it. Thanks

**Review by IntelligentQuantum — COMMENTED** [review]
`CreatureId` add ``

File: data/sql/updates/pending_db_world/rev_1641574598839293356.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1641574598839293356');
+
+-- remove ctm for creatures dalaran visitor (two fo them) by default they are just walkers but smart ai set them as runnings
+-- this corrects the issue of them flying when they land in dalaran
+DELETE FROM `creature_template_movement` WHERE  CreatureId AND (32596, 32597);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1641574598839293356.sql:5]:
```suggestion
DELETE FROM `creature_template_movement` WHERE  `CreatureID` IN (32596, 32597);
```

**Review by Kitzunu — CHANGES_REQUESTED** [review]

**Review by IntelligentQuantum — APPROVED** [review]

**Review by Kitzunu — APPROVED** [review]

**Pull request merged**